src/app/components/Curation/HierarchicalGrid/index.test.tsx

```diff
@@ -170,4 +170,31 @@ describe('Hierarchical Grid Curation', () => {
     );
     expect(container.queryByText('13 noviembre 2022')).not.toBeInTheDocument();
   });
+
+  it('should display read time when readTime is provided in summary data', () => {
+    const fixtureDataIncludingReadTime = fixture.map(fixtureSummary => ({
+      ...fixtureSummary,
+      readTime: 5,
+    }));
+
+    const container = render(
+      <HierarchicalGrid
+        headingLevel={headingLevel}
+        summaries={fixtureDataIncludingReadTime}
+        eventTrackingData={minimalEventTrackingData}
+      />,
+    );
+    expect(container.queryAllByTestId('read-time').length).toBeGreaterThan(0);
```

Comment:
Should we give the exact number here?

```suggestion
    expect(container.queryAllByTestId('read-time').length).toBe(x);
```